perun-core/src/main/java/cz/metacentrum/perun/core/impl/Utils.java

```diff
@@ -137,6 +144,69 @@ public static String join(Iterable<?> collection, String separator) {
 		return oBuilder.toString();
 	}
 
+	/**
+	 * Returns additionalUserExtSource according to the attrName
```

Comment:
This method need to be documented little more. There need to be said that it is used for purpose of synchronization form different ExtSources and that subjectFromLoginSource came from the ExtSource itself.